### PR TITLE
Make manual installers easier to open.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Allow easy copy-pasting to open the installer.

### CODE REVIEW

File: Library/Homebrew/cask/artifact/installer.rb [hunk 1]
```diff
@@ -18,9 +18,11 @@ class Installer < AbstractArtifact
...
         def install_phase(**)
           puts <<~EOS
-            To complete the installation of Cask #{cask}, you must also
-            run the installer at:
-              #{cask.staged_path.join(path)}
+            Cask #{cask} only provides a manual installer, please
+
+              open #{cask.staged_path.join(path).to_s.shellescape}
+
+            to run the installer and complete the installation.
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cask/artifact/installer.rb:25]:
```suggestion
            Cask #{cask} only provides a manual installer, please run:
              open #{cask.staged_path.join(path).to_s.shellescape}
            to complete the installation.
```

File: Library/Homebrew/cask/artifact/installer.rb [hunk 2]
```diff
@@ -18,9 +18,9 @@ class Installer < AbstractArtifact
       module ManualInstaller
         def install_phase(**)
           puts <<~EOS
-            To complete the installation of Cask #{cask}, you must also
-            run the installer at:
-              #{cask.staged_path.join(path)}
+            Cask #{cask} only provides a manual installer, please
+            open #{cask.staged_path.join(path).to_s.shellescape}
+            to run the installer and complete the installation.
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cask/artifact/installer.rb:23]:
```suggestion
            Cask #{cask} only provides a manual installer.
            To run it and complete the installation please run:
              open #{cask.staged_path.join(path).to_s.shellescape}
```

or something similar that results on just the command to run indented as the last line

**Comment by Bo98** [comment]
Just as a side note: a package manager that doesn't actually install some packages for you is a bit unusual to be honest. I assume this is sparingly used?

**Review by bevanjkay — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]
Thanks again @reitermarkus!

**Pull request merged**